server/embed/etcd.go

```diff
@@ -238,6 +231,10 @@ func StartEtcd(inCfg *Config) (e *Etcd, err error) {
 		}
 		e.tracingExporterShutdown = func() { tracingExporter.Shutdown(tctx) }
 		srvcfg.ExperimentalTracerOptions = opts
+
+		e.cfg.logger.Info(
+			"distributed tracing started",
```

Comment:
optional: are we started or enabled here?

Reply:
Depends on the sampling I guess, we register it maybe I can change it to that?